src/cljs/main/broadfcui/endpoints.cljs

```diff
@@ -582,32 +586,26 @@
        (flatten [public-map random-data])
        random-data))})
 
-(defn get-agora-namespace-acl [namespace is-conf?]
-  {:path (str "/" (if is-conf? "configurations" "methods") "/" namespace "/permissions")
+(defn persist-agora-entity-acl [config? {:keys [name namespace snapshotId]}]
+  {:path (str "/" (if config? "configurations" "methods") "/" namespace "/" name "/" snapshotId "/permissions")
+   :method :post})
+
+
+(defn get-agora-namespace-acl [namespace config?]
+  {:path (str "/" (if config? "configurations" "methods") "/" namespace "/permissions")
    :method :get})
 
-(defn post-agora-namespace-acl [namespace is-conf?]
-  {:path (str "/" (if is-conf? "configurations" "methods") "/" namespace "/permissions")
+(defn post-agora-namespace-acl [namespace config?]
+  {:path (str "/" (if config? "configurations" "methods") "/" namespace "/permissions")
    :method :post})
 
 
-(defn delete-agora-entity [config? ns n sid]
+(defn delete-agora-entity [config? {:keys [namespace name snapshotId]}]
   {:path (let [base (if config? "configurations" "methods")]
-           (str "/" base "/" ns "/" n "/" sid))
+           (str "/" base "/" namespace "/" name "/" snapshotId))
```

Comment:
in the other agora endpoints, we use a let to set base, we just make the comparison in-line. maybe change fort consistency